### PR TITLE
Add icons in ICFolderDialog

### LINKED ISSUE #399: Change add-account-button and contact button label​
As a user I want to recognize the button for adding an email account in order to  know intuitionally how to add an account. Also, I want to recognize the button for viewing contacts in order to know intuitionally how to view my contacts. ​

**Acceptance criteria​**:
- Given I am on the IMAP overview window, I see the button to add another email account ​
- Change Icon to another label of the button:​ Possibilities: ‘Add account’​
- Do not change button logic!​

_Size: XS​_

### PR DESCRIPTION
**Pull request by laugengebaeck**
This closes #399 and is currently work in progress since the contacts branch is not merged at the moment (the _add-contact_ icon would have to be moved to `ICAddressBookDialog`).

### CODE REVIEW

File: packages/IMAPClient-UI.package/ICFolderDialog.class/class/addAccountIcon.st [hunk 1]
```diff
@@ -0,0 +1,6 @@
+constants
+addAccountIcon
+	"envelope plus by Iulia Ardeleanu from thenounproject.com as 20@15.
+	Licensed under CC BY 3.0."
+	^ Form fromBinaryStream: (Base64MimeConverter mimeDecodeToBytes:
+'iVBORw0KGgoAAAANSUhEUgAAABQAAAAPCAYAAADkmO9VAAAACXBIWXMAAAOzAAADswGKfS7AAAAAGXRFWHRTb2Z0d2FyZQB3d3cuaW5rc2NhcGUub3Jnm+48GgAAAWtJREFUOI2d08tLlFEcxvHPO40J6UIjmEVRmCgILoqgRYwR5DZcRIEQCBW6bZFGl0XRhcgbEujaaOsf0qJFq1ZRLcRFgaGUVsa4OL+Jl2EuTQ/8eM97DnzPc57fORl+4RW++j8dwk0M4XMRS+jES6y3CevGs6i/ehjABfS3AesNE0dwHSegEIs/MYNJDP8DrIRHuK8mqkJuvId7GMepJrDjuIs72KpdLNT8/8EDXMK5OrA+3ArYTr3daoFQwWOUcTE3Pyx1c1qKCIpRH/GjOtFIq3iNLmxgHjfiFKcxEeAMHVKWXxoBxzGAMVzDIEalpnXjKm5jBL/xBk+xTLo2VZUwiwtNnC+GI1Ik52PciRd5h5dxJjb43gCWSbehjAPS0fdwML5ZEUcxh7WoZqoE9JvU0O0AbUrZKoblJziGKy2AnwL4HrvokTJ8K73pSoaTLSB5beMwpqRHcDaA7/AcS1kbsLwGA1pVBSv4sA8IbEgwJPNj5gAAAABJRU5ErkJggg==' readStream)
```
Comment (elenagensch) [inline packages/IMAPClient-UI.package/ICFolderDialog.class/class/addAccountIcon.st]: I would introduce some newlines likewise to addContact

File: packages/IMAPClient-UI.package/ICFolderDialog.class/class/contactsIcon.st [hunk 2]
```diff
@@ -0,0 +1,6 @@
+constants
+contactsIcon
+	"The 'users' icon taken from https://fontawesome.com/icons/users as 20@16.
+	Licensed under https://fontawesome.com/license."
+	^ Form fromBinaryStream: (Base64MimeConverter mimeDecodeToBytes:
+'iVBORw0KGgoAAAANSUhEUgAAABQAAAAQCAYAAAAWGF8bAAAACXBIWXMAAAB2AAAAdgFOeyYIAAAAGXRFWHRTb2Z0d2FyZQB3d3cuaW5rc2NhcGUub3Jnm+48GgAAAPNJREFUOI2907FKA1EQBdATY2cTe/UX1MJerExs/BZt7WwFwUr9CT/CIAELLS0lIX5ChGDQYudhXF5214AOXNh3d+7d4c5b/rnWcYs3jHGDzrJmqxjgs4QB2nXiHkaBbnBHGbOEXoWOIFLzMLjTCsOTRbqViqlflnxHjDsMHAbXls/wwXeGOV1ldRSbHQeuNdhyCjZN8IrNBh/bit6kG6HbioeNaPrAMd5xjh084REt7AX3jDOs4U5xxYTXj3wucYCZxRtOmGEfVyVef+6wrQi9ziyhj925830um8kvDCdlce4e1v5WVfqc4QWmDcym0fu39QUzEXR8pRswOgAAAABJRU5ErkJggg==' readStream)
```
Comment (elenagensch) [inline packages/IMAPClient-UI.package/ICFolderDialog.class/class/contactsIcon.st]: newlines likewise to addContact

File: packages/IMAPClient-UI.package/ICFolderDialog.class/class/addContactIcon.st [hunk 3]
```diff
@@ -1,5 +1,5 @@
 constants
-addUserIcon
+addContactIcon
```
Comment (elenagensch) [inline packages/IMAPClient-UI.package/ICFolderDialog.class/class/addContactIcon.st]: when is this icon used?

As far as I understood, we won't need the plusContact icon in ICFolderdialog

**Review by elenagensch — COMMENTED** [review]
Some clean changes, well done!

File: packages/IMAPClient-UI.package/ICFolderDialog.class/class/addContactIcon.st (hunk 3 above)
Comment (laugengebaeck) [reply packages/IMAPClient-UI.package/ICFolderDialog.class/class/addContactIcon.st]: That's right, I will move the icon as soon as the other pull request is merged (that's why this pull request was marked as a draft ;)

Comment (elenagensch) [reply packages/IMAPClient-UI.package/ICFolderDialog.class/class/addContactIcon.st]: Ok, nice

**Comment by blackjack2693** [comment]
looks great

**Review by sm1lla — APPROVED** [review]
Nice!

**Comment by laugengebaeck** [comment]
This is ready now. If somebody wants to, you can do a quick review again. Otherwise, I would merge this.

**Review by elenagensch — COMMENTED** [review]
I'm fine with merging the changes, well done!

**Pull request merged**